### ISSUE TITLE
Integrity check fails due to missing .git directory

### ISSUE THREAD

**Issue by OrelEagle** [PanCakeConnaisseur/auto_mail_accounts#9]
After an update, I got a notification telling me that some files did not pass the integrity check. I ran:

`occ integrity:check-app auto_mail_accounts`

and got the following error message:
```  
- FILE_MISSING:
    - .git/HEAD:
      - expected: 76b199f1bffdc6d21547895431b0f85f3857df400225cbd1ae55e7ded09786a8a44e1cb21f1feff4951cf57c893043a0be696dcb0e4f5758a4d6081a833eb6cf
      - current: 
    - .git/config:
      - expected: 1e788501c9ec751fd8d4043086b7daa5d53598a7f8070f20e9f095c58ae59fdf14417faf3ef03772255c3a3a0272dbc7b26c964ef86daba2c69152ecc4d8cafd
      - current: 
    - .git/description:
      - expected: f7e152f75b124c3e1c5d12f00427729d9eec4e2c1bf70d7606440a6600d003367eb178331e75ab976a61496e79c2c822020849d28ffd170946397b934611b471
      - current: 
    - .git/hooks/applypatch-msg.sample:
      - expected: 536cce804d84e25813993efdd240537b52d00ce9cdcecf1982f85096d56a521290104c825c00b370b2752201952a9616a3f4e28c5d27a5b4e4842101a2ff9bee
      - current: 
    - .git/hooks/commit-msg.sample:
      - expected: d6bb7fa747f4625adf1877f546565cbe812ca7dd4168f7e9068e6732555d8737eba549546cf5946649e3f38de82d173aaf9c160a4c9f9445655258b4c5f955eb
      - current: 
    - .git/hooks/fsmonitor-watchman.sample:
      - expected: 336ad6d610af5a6dcdf35bd5d0bddacd36bab30cebe7f20b9802e27ac04ed5a0ace7a9eb34171592b719d22cf60a3c7f4310521015dddbe5fdf4e54ae5ed3112
      - current: 
    - .git/hooks/post-update.sample:
      - expected: 473ad124642571656276bf83b9ff63ab1804d3c23a5bdae52391c6f70a894849ac60c10c9d31deff3938922ce83b68b1e60c11592bbf7ea503f4acd39968cefa
      - current: 
    - .git/hooks/pre-applypatch.sample:
      - expected: cb78aa7e9b9c146e5db65d86dd83f04e2b6942a06fab50c704a0fd900683f3b6ad1164e74afe2f267f6da91cdff0b9ab07713e12cefc6f8d741b5df194f4fda6
      - current: 
    - .git/hooks/pre-commit.sample:
      - expected: 337579c03ec152cc04240141f16a7bc471d7ea1cdc214f6e553367d03a337621e3f0ed2dc0eaa800e4a080e37b0ea3967d5daee3744b7519b5c789262beb1d39
      - current: 
    - .git/hooks/pre-merge-commit.sample:
      - expected: e4dc204494f5062efa3032b00c64707a4f38978040482501b3e085f071e3ee5a9737d537e6a52002ceb4ebe2bfe09e555c5d969581e80b3eba2a922015c67960
      - current: 
    - .git/hooks/pre-push.sample:
      - expected: 37cd20a090494acb16c2255fe93dadeb409f2553afddc409dad5b5e99c9f5612a28eba397c0f3ae771e17edbe0a6c2a3af311154285e6d9c43e4cc4f06ba5b8d
      - current: 
    - .git/hooks/pre-rebase.sample:
      - expected: 00d21d5d72386c3d9b5a1c36ba85201f730556a8295d4353af54af7892ab81010d42aff209ec1fda61c54e4dda3737cea5fda64f09d40ce5004ae28239565025
      - current: 
    - .git/hooks/pre-receive.sample:
      - expected: ee08c11fab7e896b2e09c241954ba7640338b12c75cd8040daf053c31b2f22236d7a0deac736f89d305236312fdb4f560a38d4d8debdcc9dcdd23b2d975907d5
      - current: 
    - .git/hooks/prepare-commit-msg.sample:
      - expected: 50ec8a0dd98427e80a82a8d8ce44462a845876e1594c9d0e89483ce9a8aaad616edea0e5c45c1bb69d8fe7f520c6f2260d6fa350d77b400899c3ae375e965bfb
      - current: 
    - .git/hooks/update.sample:
      - expected: 759abd33e5d1dda9b3e3d3116c47b23f616320d70cd66d567b1c4eeefeb6d78571d83b67d825d9e23aac35f0a2e0745cfab1381ecf486e30076b08797699f2c4
      - current: 
    - .git/index:
      - expected: fb1209d5fe58d344930d6071bee582c0c03ce4b9794cd49a5c7bf1ec9211db012cc2a99f9a5c63e8b9aff0b8df21d36e2d699176d24c5bdcd10048b4f0577023
      - current: 
    - .git/info/exclude:
      - expected: 9828c6ecdf91bf117416e17f4ee9caee2e1e37b6fb00b9ff04035ace17a3089b9d0a25c6baa1046c0e1c62d3da88838e8fca74ea82973d6b975905fde58f3072
      - current: 
    - .git/logs/HEAD:
      - expected: 189ffc34f5514ce2f1d2cc0641011b0b9bd92586efc3545a113ff5285d51150d1ca9cc91fafe77bd77ef92f32e811d5041ed2f3f967ba4db9d0f828eceb1cb38
      - current: 
    - .git/objects/pack/pack-dada62473010d883101d4bf942717d3f7e14fd28.idx:
      - expected: d99b2806d105d65222ce4e557050b055f1da25b743855dfa603f7445f132fdced16966fbe06cd474ac7b8fbbf700b6f0781bffc701c8c4ab5195e23f1ed7fe3e
      - current: 
    - .git/objects/pack/pack-dada62473010d883101d4bf942717d3f7e14fd28.pack:
      - expected: 557cc66ae916924c39b8e3879bb84bfa73c7cdbd6f2145d74bae00630c78b506313d1fe281c96f2c35d921734dc6dfa414d59523aef0d8f6d26c20867e18c0e0
      - current: 
    - .git/packed-refs:
      - expected: 6df90a0d45fea18457049c717dee09dbb8e562ca0ba556343a3e88e01e6bc7d28c9605375351858f61b5525187ee0b41fa04bad4a656762eec328e137e892699
      - current: 
    - .git/refs/heads/master:
      - expected: 50f183ac870bb1d63dd2b0a7b59ab0f1e3f078f3bd3a8a60b62b94ac1365d0d9fbcb9ec7c37fa68836e0a0588f9bf1981b3151388f5b90431e1e0825c5f82e35
      - current: 
    - .git/refs/remotes/origin/HEAD:
      - expected: 0d686b330d5704fdbb9588d62bbddda273f86b24db52ddbdb73c0c304aabb8e6f8a1df806b360113de6783bdc138f8731c03c22db1537cc4ad14975eb28d031f
      - current: 
```

**Comment by alexeyabel** [comment]
Thanks, included my .git folder by mistake. Will fix.

**Comment by alexeyabel** [comment]
Fixed in [v0.1.7](https://github.com/PanCakeConnaisseur/auto_mail_accounts/releases/tag/v0.1.7) Uploaded to app store already.
